Packs/Core/Playbooks/playbook-Suspicious_process_execution_by_scheduled_task_on_a_sensitive_server.yml

```diff
@@ -0,0 +1,796 @@
+id: Suspicious process execution by scheduled task on a sensitive server
+version: -1
+name: Suspicious process execution by scheduled task on a sensitive server
+description: |-
+  This playbook handles "Suspicious process execution by scheduled task on a sensitive server" alerts.
+
+  Playbook Stages:
+
+  Investigation:
+
+  During the alert investigation, the playbook will perform the following:
+
+  - Checks the suspicious process reputation.
+  - Search for related XDR agent alerts to identify any malicious activity on the server.
```

Comment:
Searches for related XDR agent alerts that finds any malicious activity on the server.